### PR TITLE
Update LLVM.org.xml

### PR DESCRIPTION
**Pull request by koops76**
#10865

### CODE REVIEW

**Comment by Bisaloo** [comment]
Also https://github.com/EFForg/https-everywhere/issues/10071.

File: src/chrome/content/rules/LLVM.org.xml [hunk 1]
```diff
@@ -30,24 +27,12 @@ Fetch error: http://www.reviews.llvm.org/ => https://www.reviews.llvm.org/: (6,
...
-
-			<!--	+ve:
-					-->
-			<test url="http://this.host.reviews.llvm.org/" />
-			<test url="http://exists.not.reviews.llvm.org/" />
-
-		<test url="http://www.reviews.llvm.org/" />
 
 	<!--	Complications:
 				-->
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/LLVM.org.xml]: It seems like the complication rules can be replaced with direct rewriting rules now

Comment (koops76) [inline src/chrome/content/rules/LLVM.org.xml]: @gloomy-ghost Done. Do not merge yet, I would like to run Sublist3r.

Comment (koops76) [inline src/chrome/content/rules/LLVM.org.xml]: @gloomy-ghost Done.

**Comment by gloomy-ghost** [comment]
Thanks!

**Pull request merged**